webapp/src/main/webapp/src/app/groups/import/group-import.component.ts

```diff
@@ -35,6 +36,16 @@ export class GroupImportComponent implements OnInit {
 
     this.uploader.onCompleteAll = () => {
       this.uploader.clearQueue();
+    };
+
+    window.onbeforeunload = this.confirmNavigation.bind(this);
```

Comment:
This will guard against navigation outside the angular app (i.e. tab close or url change)